flask_appbuilder/security/manager.py

```diff
@@ -1054,7 +1056,8 @@ def auth_user_oauth(self, userinfo):
             if not user:
                 log.error("Error creating a new OAuth user %s" % userinfo["username"])
                 return None
-        self.update_user_auth_stat(user)
+        if user:
```

Comment:
We have already made sure the user exists here, or not?

Reply:
true